tools/dotnet-linker/Steps/StoreAttributesStep.cs

```diff
@@ -21,6 +21,13 @@ protected override void ProcessAttribute (ICustomAttributeProvider provider, Cus
 					break;
 				}
 				break;
+			case "ObjCRuntime":
+				switch (attr_type.Name) {
+				case "BindingImplAttribute":
+					store = true;
```

Comment:
weird that this has not caused any issue for iOS, tvOS and Catalyst :|

Reply:
This might have been a mistake on my part, CI revealed that the test in question still fails, and I found another issue that explains.